File: app/src/main/java/my/edu/tarc/communechat_v2/LocalDatabase/AppDatabase.java
package my.edu.tarc.communechat_v2.LocalDatabase;

import android.arch.persistence.room.Database;
import android.arch.persistence.room.RoomDatabase;
import android.arch.persistence.room.TypeConverters;

import my.edu.tarc.communechat_v2.model.Chat_Room;
import my.edu.tarc.communechat_v2.model.Message;

@Database(entities = {Message.class, Chat_Room.class}, version = 1)
@TypeConverters({Converters.class})
public abstract class AppDatabase extends RoomDatabase {
    public abstract ChatRoomDao chatRoomDao();

    public abstract MessageDao messageDao();
}

File: app/src/main/java/my/edu/tarc/communechat_v2/LocalDatabase/AppDatabase.kt
package my.edu.tarc.communechat_v2.LocalDatabase

import android.arch.persistence.room.Database
import android.arch.persistence.room.RoomDatabase
import android.arch.persistence.room.TypeConverters

import my.edu.tarc.communechat_v2.model.Chat_Room
import my.edu.tarc.communechat_v2.model.Message

@Database(entities = [Message::class, Chat_Room::class], version = 1)
@TypeConverters(Converters::class)
abstract class AppDatabase : RoomDatabase() {
    abstract fun chatRoomDao(): ChatRoomDao

    abstract fun messageDao(): MessageDao
}
